### PR TITLE
[IN-2762] Fix env paths

### PR DESCRIPTION
**Pull request by nagypeterjob**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [X] I updated the corresponding tests if there were any.
- [X] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [X] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [X] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/profiles/files/bashrc [hunk 1]
```diff
@@ -16,6 +16,10 @@ export GOPATH="$HOME/go"
 export PATH="$PATH:$GOPATH/bin"
 export GO111MODULE=auto
 
+# Flutter & fvm
+export PATH="$PATH:$HOME/.pub-cache/bin"
```
Comment (bitrise-tom) [inline roles/profiles/files/bashrc:20]: we should add the same paths to the zshrc too

Comment (nagypeterjob) [reply roles/profiles/files/bashrc:20]: Good catch!

**Review by fehersanyi-bitrise — APPROVED** [review]
yolo

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Review by fehersanyi-bitrise — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request merged**